core/api/serializers/project_v2.py

```diff
@@ -562,3 +575,19 @@ def _update_or_create_ods_odp(self, instance, ods_odp_data):
 
         if ids_to_delete:
             instance.ods_odp.filter(id__in=ids_to_delete).delete()
+
+    def _create_history(self, project, request_user):
```

Comment:
I would define the same method (e.g. `_log_history(self, project, request_user, description)`) to be used in both *create* and *update* because the action on **ProjectHistory** is the same(*create*) and I believe it is clearer this way.